locale/de/LC_MESSAGES/django.po

```diff
@@ -3894,15 +3907,33 @@ msgid ""
 "If this was not you, it is safe to disregard this email.\n"
 msgstr ""
 
-#: .\course\utils.py:337
+#: .\course\templates\course\task-monitor.html:12
+#, fuzzy
```

Comment:
Just for your reference, you need to find all blocks with ``#, fuzzy`` lines, remove those lines and adjust your translations, when the django.po files are modified by ``makemessages`` command. Or those literals won't be translated.